src/types/algoliasearch.ts

```diff
@@ -16,15 +16,27 @@ import algoliasearch, {
   SearchForFacetValues as SearchForFacetValuesV3,
 } from 'algoliasearch';
 import {
+  FindAnswersResponse as FindAnswersResponseV4,
   SearchResponse as SearchResponseV4,
+  FindAnswersOptions as FindAnswersOptionsV4,
   // no comma, TS is particular about which nodes expose comments
   // eslint-disable-next-line prettier/prettier
-  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4
+  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4,
   /** @ts-ignore */
   // @ts-ignore
   // eslint-disable-next-line import/no-unresolved
```

Comment:
These lines don't anything for me anymore, is that normal?

Reply:
If you run these two lines, it will fail without the line:
https://github.com/algolia/instantsearch.js/blob/feat/answers/.circleci/config.yml#L70:L71